keyboards/tada68/keymaps/hhkb68/README.md

```diff
@@ -1,2 +1,2 @@
 # HHKB68
```

Comment:
This file should be renamed to fully lowercase `readme.md`. We are trying to enforce lowercase filenames wherever possible, but you are correct that this was missed during review.